biomage/stage/stage.py

```diff
@@ -431,12 +434,7 @@ def paginate_experiments(
                 answers.remove(done_text)
                 page_action = "done"
 
-            chosen_experiments = set(
-                [
-                    *chosen_experiments,
-                    *[experiment.split(" ")[0] for experiment in answers],
-                ]
-            )
+            chosen_experiments = {*chosen_experiments, *[experiment.split(" ")[0] for experiment in answers]}
```

Comment:
why did you remove the set?

Reply:
It's still a set, I just used a set literal as it's faster and shorter.

Reply:
oh right, hadn't seen that notation 👌